### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by corkage-waltzes**

### CODE REVIEW

**Review by charity-answer — COMMENTED** [review]
Need more content in README

**Review by charity-answer — COMMENTED** [review]
Add some examples in README

**Comment by corkage-waltzes** [comment]
README.md

MOre please

File: README.md
Comment (corkage-waltzes) [inline README.md:1]: Need more description

Comment (corkage-waltzes) [inline README.md:1]: Need more examples

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# experimental
```
Comment (charity-answer) [inline README.md:1]: Add more description

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
\ No newline at end of file
+# experimental
+
```
Comment (charity-answer) [inline README.md:2]: Add stuff

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
...
+There will be some important notes in here
```
Comment (charity-answer) [inline README.md:3]: Finally address this

**Pull request merged**